src/puppet-web/puppet-web.ts

```diff
@@ -437,7 +450,7 @@ export class PuppetWeb extends Puppet {
     })
     if (!mediaId)
       throw new Error('upload fail')
-    return mediaId as string
+    return { mediaId, mediaData }
```

Comment:
Could you add mediaId into mediaData?

like:

```ts
Object.assign(mediaData, {
  mediaId.
})
```

If we could do this, then here will be more clear, because we can return the interface of `MediaData` here.